spells.yml

```diff
@@ -9860,6 +9882,32 @@ wall_of_fire:
     - fire
     - sorcerer
     - wizard
+wall_of_flesh:
+  casting_time: 1 standard action
+  components: V, S, M
+  concentration: true
+  description: |-
+    You create a wall of living flesh within range. The wall is 1 foot thick and is composed of ten 10-foot-by-10-foot panels. Each panel must be contiguous with at least one other panel, be vertically oriented, and be supported underneath by a solid surface or on any sides by at least two other panels. If the wall cuts through a creature's space when it appears, the creature is pushed to one side of the wall (your choice).
+
+    Each panel has 10 AC and 30 hit points. A panel is destroyed if it is reduced to 0 hit points, potentially causing other panels to collapse and be destroyed.
```

Comment:
Given that we're bringing grappled into this, I do think it would be worthwhile to explicitly give each panel a size of Large.